scripts/ksql/run_ksql_commands.sh

```diff
@@ -3,5 +3,5 @@
 input=/tmp/ksqlcommands
 while IFS= read -r line
 do
-  echo -e "$line\nexit" | ksql -u ksqlUser -p ksqlUser http://ksql-server:8088
+  echo -e "$line\nexit" | ksql -u ksqlUser -p ksqlUser http://ksqldb-server:8088
```

Comment:
Should the username change to ksqldbUser?

Reply:
Yes, it should